components/filter-field/src/filter-field-tag/filter-field-tag.ts

```diff
@@ -88,7 +110,37 @@ export class DtFilterFieldTag {
     }
   }
 
-  constructor(private _changeDetectorRef: ChangeDetectorRef) {}
+  /** @internal Element reference to the tag that holds the value. */
+  @ViewChild('valueSpan', { static: false, read: ElementRef })
+  _valueSpan: ElementRef<HTMLSpanElement>;
+
+  /** @internal Configuration object for the tag overlay */
+  _overlayConfig: DtOverlayConfig = {
+    pinnable: false,
+    originY: 'edge',
+  };
+
+  /** @internal Flag that determines if the tooltip needs to be shown. */
+  _tooltipDisabled = false;
+
+  constructor(
+    private _changeDetectorRef: ChangeDetectorRef,
+    private _platform: Platform,
+    _zone: NgZone,
+  ) {
+    this._stateChanges$
+      .pipe(
+        takeUntil(this._destroy$),
```

Comment:
this cannot work `takeUntil` has to be the last pipeable operator when you perform a `switchMap` afterwards the `takeUntil` will not notify.